source/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -43,14 +43,16 @@ Prerequisites
 Download the |CL| disk image for Hyper-V
 ****************************************
 
-#. Download the :file:`clear-[VERSION]-hyperv.vhdx.gz` for Microsoft* Hyper-V
-   from the `downloads`_ website.
+#. Download the :file:`clear-[VERSION]azure-hyperv.vhd.gz` for Microsoft* 
+   Hyper-V from the `downloads`_ website.
 
 #. Verify and extract the image using these instructions:
    :ref:`download-verify-decompress`.
 
-   After extraction, the file should be named :file:`clear-[VERSION]-hyperv.vhdx`.
+   After extraction, the file should be named :file:`clear-[VERSION]-hyperv.vhd.gz`.
```

Comment:
After extration, the file doesn't have a gz extension.  Also probably want to call this out here as `clear-[VERSION]-azure-hyperv.vhd`

Reply:
Yes. Removed `.gz`